### DISCUSSION TITLE
Tab Groups

### DISCUSSION THREAD

**Discussion by Hate9** [pulse-browser/browser#276] [Ideas]
I know addons like Simple Tab Groups exist, but I'd really love a more integrated tab group option, like how Opera does (although without the weird limit on the number of groups you can have). I like Firefox a lot, but the lack of a fully-integrated tab groups option is still my biggest gripe.

**Comment by trickypr** [comment]
I have been thinking about this a fair amount, and I don't think that the way that pulse is structured would allow me to implement it to the level of quality that I want. You might want to keep an eye on [Dot](https://github.com/dothq). They are still prototyping, but it might be interesting in a few years time.

**Reply by ioc2e3** [reply]
but ...

about Dot Browser?

The latest version is (as of July 25, 2021 ) ...
![1rr](https://github.com/pulse-browser/browser/assets/152480314/6e5c16f3-5266-4ca9-8470-d711674f4b4c)


What should I do? Keep waiting? Or?

.
